docs/build.jl

```diff
@@ -16,15 +16,12 @@ cd(dirname(@__FILE__)) do
     end
 
     # Generate and save the contents of docstrings as markdown files.
-    for m in modules
-        filename = joinpath(api_directory, "$(module_name(m)).md")
-        try
-            save(filename, m)
-        catch err
-            println(err)
-            exit(1)
-        end
+    index  = []
```

Comment:
`[]` produces an empty `0-element Array{None,1}` in 0.3 and `0-element Array{Any,1}` in 0.4, the first isn't resizeable. You can get around that with `Any[]` in both versions.